src/services/functionAppService.ts

```diff
@@ -43,22 +51,11 @@ export class FunctionAppService extends BaseService {
     return response.data.value;
   }
 
-  /**
-   * Initialize deployment artifact container if rollback is specified
-   */
-  public async initialize() {
```

Comment:
Is this unnecessary because of the added params in the constructor?

Reply:
Rather than initializing the blob storage here, it initializes before actually creating the blob. The problem was it was trying to initialize and create a container for an account that didn't yet exist